tests/integration.rs

```diff
@@ -84,6 +84,24 @@ fn read_root_with_neg_serial() {
     let _ = webpki::TrustAnchor::try_from_cert_der(ca).expect("idcat cert should parse as anchor");
 }
 
+#[test]
+fn read_ee_with_neg_serial() {
+    let ca: &[u8] = include_bytes!("misc/serial_neg_ca.der");
+    let ee: &[u8] = include_bytes!("misc/serial_neg_ee.der");
+
+    let anchors = vec![webpki::TrustAnchor::try_from_cert_der(ca).unwrap()];
+    let anchors = webpki::TLSServerTrustAnchors(&anchors);
+
+    //#[allow(clippy::unreadable_literal)] // TODO: Make this clear.
```

Comment:
Just a note to other reviewers: this TODO is not unique to this work, but another instance of a pattern established by 2 other tests in this file. It would be nice to fix these up but I don't think it needs to be done in this branch.